docs/content/en/tasks/repair/_index.md

```diff
@@ -7,27 +7,26 @@ description: "Use the Reaper for Apache Cassandra&reg; web interface to perform
 Repairs are a critical anti-entropy operation in Cassandra. In the past, there have been many custom solutions to manage them outside of your main Cassandra installation. K8ssandra Operator provides the Reaper web interface that eliminates the need for a custom solution. Just as K8ssandra Operator makes Cassandra setup easy, Reaper makes configuration of repairs even easier.
 
 {{% alert title="Tip" color="success" %}}
-The requirement for your environment may vary considerably, however the general recommendation is to run a repair operation on your Cassandra clusters about once a week.
+The requirement for your environment may vary considerably, however the general recommendation is to run a repair operation on your Cassandra clusters once a week.
 {{% /alert %}}
 
 ## Tools
 
 * Web Browser
-* `values.yaml` configuration, or use `--set` flags on the command line
+* kubectl
 
 ## Prerequisites
 
 1. Kubernetes cluster with the following elements deployed:
    * If you haven't already installed a K8ssandraCluster using K8ssandra Operator, see the [local install]({{< relref "/install/local" >}}) topic.
-   * [Ingress Controller]({{< relref "tasks/connect/ingress" >}})
-2. DNS name configured for the Reaper web interface
 
-{{% alert title="Tip" color="success" %}}
-As an alternative to configuring an Ingress, consider **port forwarding**. It's another way to provide external access to resources that have been deployed by K8ssandra Operator in your Kubernetes environment. Those resources could include Prometheus metrics, pre-configured Grafana dashboards, and the Reaper web interface for repairs of Cassandra data. The `kubectl port-forward` command does not require an Ingress/Traefik to work. 
+Access to the Reaper web interface requires either:  
 
-* Developers, see [Set up port forwarding]({{< relref "/quickstarts/developer/#set-up-port-forwarding" >}}).  
-* Site reliability engineers, see [Configure port forwarding]({{< relref "/quickstarts/site-reliability-engineer/#port-forwarding" >}}).
-{{% /alert %}}
+* setting up a custom Ingress resource
+* or modifying Reaper's Kubernetes service into a LoadBalancer (in cloud environments), which will expose it over a public IP
```

Comment:
Suggestion - "or modifying Reaper's Kubernetes service to function **as a** LoadBalance (in cloud environments), ... "

Reply:
I made the change 👍